src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -248,6 +248,16 @@ protected void submit(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
         return triggers.toMap();
     }
 
+    public void addTrigger(Trigger trigger) {
+        Trigger old = triggers.get(trigger.getDescriptor());
+        if (old != null) {
+            old.stop();
+            triggers.remove(old);
+        }
+        triggers.add(trigger);
+        trigger.start(this, true);
```

Comment:
> Should it be explicitly saved here?

`triggers` is a `DescribableList` and thus a `PersistedList`.

> /me wonders about [a hypothetical?] `@PersistsData` annotation

IMO annotations are of little value unless there is code which can somehow interpret them, at least during static analysis.